plugins/webhooks/helper.js

```diff
@@ -462,13 +498,11 @@ async function createPREvents(options, request) {
                 let subscribedConfigSha = '';
                 let eventConfig = {};
 
-                // Check if the webhook event is from a subscribed repo and
-                // and fetch the source repo commit sha and save the subscribed sha
                 if (uriTrimmer(scmConfig.scmUri) !== uriTrimmer(p.scmUri)) {
                     subscribedConfigSha = sha;
 
                     try {
-                        sha = await pipelineFactory.scm.getCommitSha({
+                        configPipelineSha = await pipelineFactory.scm.getCommitSha({
```

Comment:
`configPipelineSha` is not defined

Reply:
it is initialized https://github.com/screwdriver-cd/screwdriver/blob/c3d7ce4a9a22a2e98233fcf85c3245d774f4ede2/plugins/webhooks/helper.js#L497